beesdoo_product/models/beesdoo_product.py

```diff
@@ -219,15 +226,9 @@ def generate_barcode(self):
 
     @api.multi
     @api.depends("seller_ids", "seller_ids.date_start")
-    def _compute_main_seller_id(self):
+    def _compute_main_supplierinfo_id(self):
         for product in self:
-            # todo english code Calcule le vendeur associé qui a la date de
-            #  début la plus récente et plus petite qu’aujourd’hui fixme
-            #   could product.main_seller_id be used instead? it seems that
-            #   “seller” and “supplier” are used interchangeably in this
-            #   class. is this on purpose?
-            sellers_ids = product._get_main_supplier_info()
-            product.top_supplierinfo_id = sellers_ids and sellers_ids[0] or False
+            product.main_supplierinfo_id = product._get_main_supplier_info()
```

Comment:
Comments are now in the `_get_main_supplier_info()` method.

Reply:
No need to double check that there is only one supplierinfo as it is part of the `_get_supplierinfo()` method.